bazel_tools/proto.bzl

```diff
@@ -194,7 +195,9 @@ def _proto_scala_deps(grpc, proto_deps):
     ] if grpc else []) + [
         "%s_scala" % label
         for label in proto_deps
-    ]
+    ] + ([
+        "@maven//:io_grpc_grpc_services",
+    ] if java_conversions else [])
```

Comment:
Fix 3: adding a compile-time dependency that we need when building with Java conversions.